config/checker/com.intellij.astub

```diff
@@ -230,6 +230,18 @@ class WriteAction<T> {
 @UIPackage package com.intellij.openapi.fileChooser;
 @UIPackage package com.intellij.openapi.fileEditor;
 
+class FileEditorManager {
+
+  @SafeEffect
```

Comment:
All methods in an `@UIPackage` (same for `@UI` class) are considered `@UIEffect` unless stated otherwise.